### PR TITLE
Modified the fluff of the loyalty implan to better fit

### PR DESCRIPTION
**Pull request by Robertha89**
Updating ingame effects of loyalty implants according to
http://nanotrasen.se/forum/viewtopic.php?f=10&t=4078
Removing vampires being hurt by holy water and sunlight, all it does is
nullifies their power, added to put it on the same line as
http://nanotrasen.se/forum/viewtopic.php?f=48&t=4391

### CODE REVIEW

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 1]
```diff
@@ -445,7 +445,7 @@ datum
 			reagent_state = LIQUID
 			color = "#0064C8" // rgb: 0, 100, 200
 
-			on_mob_life(var/mob/living/M as mob)
+			/*on_mob_life(var/mob/living/M as mob)
```
Comment (veganzombeh) [inline code/modules/reagents/Chemistry-Reagents.dm]: Am I mistaken, or is this block comment not closed?

**Comment by tigercat2000** [comment]
Uh. This seems pointless, honestly.

**Comment by SamCroswell** [comment]
Travis failed.
Also, the vampires are only hurt by ingested Holy Water. This was by design. Highly disagree with that being removed.

**Comment by Fox-McCloud** [comment]
This also makes revs impossible to counter; unless the head rev is BLATANTLY caught in the act of flashing someone, he'll never be found.

Not only that, but if a few plants go around carrying flashes to to fake them out--security will literally be powerless to stop revolutions other than no holds bar executing anyone and everyone with even remotely suspicious behavior connected with revolutionary activity.

**Comment by veganzombeh** [comment]
I'm against this. It's a massive buff to vampires and revs. Although, I would :+1: the cosmetic changes (name, desc, etc) to the loyalty implant.

**Comment by TheDZD** [comment]
DNM until compile errors are fixed.

**Comment by Robertha89** [comment]
There we go, should work now, just waiting for travis to check out.
Revs and cultist can resist implants, with a chance of a normal guy getting implanted also resisting as a anti-meta tool.

**Comment by tigercat2000** [comment]
@Robertha89 You mean as a way to get innocent people executed and cause a shitstorm for admins every time the loyalties are brought out?

I didn't look at the files until now, and in my opinion all of this will do a lot more harm than good. This leaves beating people's faces in as the only way to deconvert them, thus increasing they already fucking chaotic mess that is revolution. Cultists, uh, why? So people can actually check cultists past when they get a book, and then proceed to execute an innocent because they could also show up as resisting?

As for the vampire stuff, why? They are fine the way they are, I don't see why you would make them immune to starlight or make holy water utterly useless.

**Comment by Robertha89** [comment]
@tigercat2000 the fact you can essentially cuff everyone, inject them with holy water or drag them to chapel, is the same reason for why the changeling test was removed. 

Regarding the loyalty implants, you can still ditch your flash, or your book, and deconverting them by beatings has not been in the code for ages. 
With anyone being able to resist implants, does not mean that if they resist they are an antag. Just like if i carry around a tome, or any typical syndicate gear does not mean I am an antag either, at best it's contraband, and killing because antag is, and has always been a bad way of going about with it. 

As for the vampire stuff, why does startlight even harm them other then lol vampires burn in sunlight, can just as well replace it with sparkling in starlightt instead of burning to death from something you can't track, and holy water is not useless, can do the can't cult people with holy water inside of them. It still removes a vampires power, unless they are full power.

The reason for why is simple, this will reduce the amount of powergameing, because there is a failure chance in something, it will also be anti-meta, you know like so many other things

**Comment by Fox-McCloud** [comment]
>>regarding loyalty implants, you can still ditch your flash, or book, and deconverting them by beatings >>has not been in the code for ages.

WHut: https://github.com/ParadiseSS13/Paradise/blob/bb9a6c5a81d2f2e13bfbcbf252e83b6a3614a59d/code/modules/mob/living/carbon/human/human_defense.dm#L310

**Comment by Earthdivine** [comment]
As the admin who headed a lot of these changes, vampires ingesting holy water takes A LONG time for it to start hurting them, to the point your far better off just dragging them to the chapel. The Holy water and changling test changes were curbed to stop Yolotesting, not testing in general. 

As for rev's, removing this is TERRIBLE. Not from a detecting or valid detection sense. There would be NOTHING to keep a head rev from immediately reflashing the person after you deconvert them. 

I am totally :+1: to renaming the implants, as I wanted to do that but could never get GitHub to work, but the gameplay changes need a lot more carefulness, particullarly taking out the most major part of a game mode.

Also:

Bashingproof

**Comment by ZomgPonies** [comment]
:-1:  This was an ill-thought out PR.

**Comment by Robertha89** [comment]
everything should be so only the implant name and description part is changed.

File: code/game/gamemodes/vampire/vampire.dm [hunk 2]
```diff
@@ -182,7 +182,7 @@
 	if (you_are)
 		dat = "<B>\red You are a Vampire! \black</br></B>"
 	dat += {"To bite someone, target the head and use harm intent with an empty hand. Drink blood to gain new powers.
-You are weak to holy things and starlight. Don't go into space and avoid the Chaplain, the chapel and especially Holy Water."}
+You are weak to holy things. Avoid the Chaplain, the chapel and Holy Water."}
```
Comment (Fox-McCloud) [inline code/game/gamemodes/vampire/vampire.dm]: please return the original text so people aren't asking "why am I burning in space?"

**Comment by tigercat2000** [comment]
To be honest. I don't like renaming it to "brainwash prevention". It feels dull.
Just my opinion on it.

**Comment by Fox-McCloud** [comment]
I admit I'm not a big fan either---this would have made sense had it immediately followed our change to loyalty implants--but players have had a few months now to adjust to the new mechanics; this just seems like it would cause further confusion.

**Comment by Keilden** [comment]
People still think implants makes you loyal.

**Comment by veganzombeh** [comment]
I'd argue that the Important Notes and Special Features of the implant should be changed, but the name should remain the same for the reasons Fox said.

**Pull request closed without merging**